metrics/timescaledb/db.py

```diff
@@ -11,10 +11,9 @@
 log = structlog.get_logger()
 
 
-def delete_rows(connection, name, n=10000):
-    """
-    Delete N rows from the given table
-    """
+def delete_rows(connection, name, batch_size):
+    if not batch_size:
+        batch_size = 10000
```

Comment:
When I first looked at this I wasn't sure why it wasn't

```suggestion
def delete_rows(connection, name, batch_size=10000):
```

I then looked at how the function was used and decided that `batch_size=None` communicated that there was no limit to the batch size (and made the obligatory "There's no limit!" joke). But on reflection this isn't quite correct. Unless 10,000 is, in effect, a large enough number that it's indistinguishable from no limit?

Reply:
No idea what I was thinking here. This is just a mistake. Thanks.